src/ui-test/camelk_java_debug_test.ts

```diff
@@ -0,0 +1,197 @@
+import * as uiTestConstants from './utils/uiTestConstants';
+import { promiseFail, promiseSucceed } from './utils/promiseUtils';
+
+const TEST_FOLDER = '../../../testFolder';
+const WORKSPACE_FOLDER = path.join(__dirname, TEST_FOLDER);
+
+const START_DEBUG_LABEL = uiTestConstants.startDebug;
+const REMOVE_INTEGRATION_LABEL = uiTestConstants.integrationRemove;
+
+describe.only('Test Debug on Camel K Integrations from Side Bar', function () {
+
+	before(async function () {
+		this.timeout(90000);
+		this.retries(3);
+		await VSBrowser.instance.waitForWorkbench;
+		await prepareTempWorkspaceForTests(WORKSPACE_FOLDER);
+		await VSBrowser.instance.driver.wait(camelKToolingIsEnabled);
+	});
+
+	describe('Java Debug', function () {
+		
+		const INTEGRATION_LABEL = 'java-debug-test';
+		const INTEGRATION_FILE = 'JavaDebugTest';
+
+		before(async function (){
+			this.timeout(60000);
+			await moveToExplorerActivity();
+			await createIntegration(INTEGRATION_FILE);
+			await startIntegration(INTEGRATION_LABEL);
+		})
+
+		it('Check Java Debug available', async function () {
+			this.timeout(20000);
+
+			const item = await getIntegration(INTEGRATION_LABEL);
+			const menu = await item.openContextMenu();
+			await promiseSucceed(VSBrowser.instance.driver.wait(() => menu.hasItem(START_DEBUG_LABEL), 5000));
+		});
+
+		after(async function() {
+			this.timeout(20000);
+			await removeIntegration(INTEGRATION_LABEL);
+			await prepareEmptyTestFolder(WORKSPACE_FOLDER);
+		});
+
+	});
+
+	describe('No Java Debug on Invalid Files', function() {
+
+		const INTEGRATION_LABEL = 'java-debug-test-invalid';
+		const INTEGRATION_FILE = 'JavaDebugTestInvalid';
+
+		before(async function (){
+			this.timeout(60000);
+			await moveToExplorerActivity();
+			await createIntegration(INTEGRATION_FILE);
+			await modifyCurrentFileToBeInvalid();
+			await startIntegration(INTEGRATION_LABEL);
+		});
+
+		it('Test Java Debugger Not Available On Invalid File', async function() {
+			this.timeout(20000)
+			const item = await getIntegration(INTEGRATION_LABEL);
+			const menu = await item.openContextMenu();
+			await promiseFail(VSBrowser.instance.driver.wait(() => menu.hasItem(START_DEBUG_LABEL), 5000));
+		});
+
+		after(async function() {
+			this.timeout(20000);
+			await removeIntegration(INTEGRATION_LABEL);
+			await prepareEmptyTestFolder(WORKSPACE_FOLDER);
+		});
+	})
+
+});
+
+async function prepareTempWorkspaceForTests(workspaceFolder: string) {
+	await new EditorView().closeAllEditors();
+	await prepareEmptyTestFolder(workspaceFolder);
+	await VSBrowser.instance.openResources(workspaceFolder);
+}
+
+async function camelKToolingIsEnabled() {
+	const marketplace = await Marketplace.open();
+	const item = await marketplace.findExtension(`@installed ${pjson.displayName}`);
+	return extensionIsActivated(item);
+}
+
+async function moveToExplorerActivity() {
+	const ACTIVITY_LABEL = 'Explorer';
+
+	return await moveToActivity(ACTIVITY_LABEL);
```

Comment:
redundant await (reported by Sonar)
```suggestion
	return moveToActivity(ACTIVITY_LABEL);
```